bucket/rednotebook.json

```diff
@@ -0,0 +1,22 @@
+{
+    "homepage": "https://rednotebook.sourceforge.io/",
+    "description": "A modern journal",
+    "license": "GPL-2.0-or-later",
```

Comment:
- <https://github.com/jendrikseipp/rednotebook/blob/80d6e151b3a289a4bede512cd0b55470a8d5a88a/data/rednotebook.appdata.xml#L5>
- > RedNotebook is published under the GPLv2+. Since it bundles code released under the LGPLv3+, the resulting work is licensed under the GPLv3+

Reply:
I'm no expert about the licence and happy to change it